python/iceberg/core/base_snapshot.py

```diff
@@ -127,7 +126,7 @@ def __repr__(self):
     def __str__(self):
         return self.__repr__()
 
-    def get_filtered_manifest(self, path, part_filter, row_filter, columns):
-        reader = ManifestReader.read(self.ops.new_input_file(path))
+    def get_filtered_manifest(self, path, part_filter=None, row_filter=None, columns=None):
```

Comment:
how come the `None` were removed?

Reply:
Hi , I didn't remove but rather added None to be the default argument for (part_filter, row_filter, column)

They actually are not used in method itself.. but not sure if anyone was calling with those them.  I can just remove them as well?

Reply:
Doh! Misread the diff. I like your way.